### PR TITLE
changed collection arg to argregate on 2.4 network modules

### PR DESCRIPTION
**Pull request by newswangerd**
##### ISSUE TYPE

 - Network Pull Request

##### COMPONENT NAME

- network modules

##### ANSIBLE VERSION

```
ansible 2.4.0 (collection-to-agregate 0eb6ecb932) last updated 2017/07/11 11:17:03 (GMT -400)
  config file = None
  configured module search path = [u'/Users/dnewswan/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/dnewswan/code/ansible/lib/ansible
  executable location = /Users/dnewswan/code/ansible/bin/ansible
  python version = 2.7.13 (default, Apr  4 2017, 08:47:57) [GCC 4.2.1 Compatible Apple LLVM 8.1.0 (clang-802.0.38)]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: lib/ansible/modules/network/eos/eos_user.py [hunk 1]
```diff
@@ -315,7 +315,7 @@ def main():
     """ main entry point for module execution
     """
     argument_spec = dict(
-        users=dict(type='list', aliases=['collection']),
```
Comment (privateip) [inline lib/ansible/modules/network/eos/eos_user.py:318]: change argument to `aggregate` and add alias to `users`

File: lib/ansible/modules/network/junos/junos_user.py [hunk 2]
```diff
@@ -250,7 +250,7 @@ def main():
     """ main entry point for module execution
     """
     argument_spec = dict(
-        users=dict(type='list', aliases=['collection']),
+        users=dict(type='list', aliases=['collection', 'aggregate']),
```
Comment (privateip) [inline lib/ansible/modules/network/junos/junos_user.py]: swap `aggregate` and `users` so the argument is `aggregate` and the alias is `users`

File: lib/ansible/modules/network/nxos/nxos_user.py [hunk 3]
```diff
@@ -298,7 +298,7 @@ def main():
     """ main entry point for module execution
     """
     argument_spec = dict(
-        users=dict(type='list', no_log=True, aliases=['collection']),
+        users=dict(type='list', no_log=True, aliases=['collection', 'aggregate']),
```
Comment (privateip) [inline lib/ansible/modules/network/nxos/nxos_user.py]: same as earlier about swapping

**Review by privateip — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/network/eos/eos_user.py (hunk 1 above)
Comment (newswangerd) [reply lib/ansible/modules/network/eos/eos_user.py:318]: I made the change. Not sure why github isn't registering it.

**Comment by newswangerd** [comment]
need to change collections in tests

**Comment by newswangerd** [comment]
Can't test the following modules because tests are not present/enabled or already failing:
* ios_user
* junos_user
* junos_interface
* junos_logging
* junos_static_route
* junos_vlan
* iosxr_user
* nxos_user
* vyos_user
* vyos_interface
* vyos_l3_interface
* net_interface
* net_linkagg
* net_l2_interface
* net_vlan
* net_l3_interface
* net_vrf
* net_static_route
* net_linkagg
* net_user

**Pull request merged**